wiki/iNiS_games/id.md

```diff
@@ -0,0 +1,35 @@
+---
+stub: true
+tags:
+  - DS
+  - EBA
+  - Elite Beat Agents
+  - iNiS Corporation
+  - Nintendo
+  - Nintendo DS
+  - original game
+  - original games
+  - Osu! Tatakae! Ouendan!
+  - Osu Tatakae Ouendan
+  - Ouendan
+---
+
+# Gim dari iNiS
+
+[Jenis permainan](/wiki/Game_Modes) [osu!standard](/wiki/Game_Modes/osu!) dan bahkan osu! sendiri berdasar pada gim ritme yang dikembangkan oleh **[iNiS](https://en.wikipedia.org/wiki/INiS "iNiS di Wikipedia")** untuk [Nintendo DS](https://en.wikipedia.org/wiki/Nintendo_DS "Nintendo DS di Wikipedia") pada awal tahun 2000-an. ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) mengembangkan osu! untuk menjadi simulator dan editor level untuk gim - gim yang dikembangkan oleh iNiS, dan perlu diketahui anggota komunitas osu! paling awal saat itu pada umumnya juga merupakan penggemar gim tersebut. [Beatmap](/wiki/Beatmaps) yang dibuat pada masa itu sangat dipengaruhi oleh desain level yang dibuat oleh iNiS.
```

Comment:
Only this wikipedia article that has bahasa Indonesia translation

https://id.wikipedia.org/wiki/Nintendo_DS